assets/src/js/Components/ArchivePicker/ArchivePicker.js

```diff
@@ -293,22 +298,32 @@ export default function ArchivePicker({view = ADMIN_VIEW}) {
     }
   }, []);
 
+  const processImageForBlock = async (imageID, updateAttributeFunction) => {
```

Comment:
Yes, but we would have to prevent the `getImageDetails()` method from running which in that case means we will need to run another check within the function just for the Happypoint block. So my thinking was run the check once and use the function for only blocks that follow same steps.

What do you think?

Reply:
Yeah I see your point, makes sense 🙂